### DISCUSSION TITLE
Buttom to link existing alert to a panel

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73234] [Alerting]
When we create a new alert through the `Alert` tab in a Panel, it automatically gets the `Panel ID` and `Dashboard UID` annotations.

If we have a standalone alert that was created through `Alerting > Alert Rules > New Alert`, it lacks those annotations and we have to manually insert them (which is error-prone and inefficient).

The `Alert` tab in the panel configuration should have a button similar to `Create alert rule from this panel` that said `Link existing alert to this panel` or something similar. We could select the alert from a search or dropdown menu.

**Comment by armandgrillet** [comment]
Great idea, related GitHub issue where the discussion will continue: https://github.com/grafana/grafana/issues/73914
